### PR TITLE
Feat(frontend): add missing rules actions

### PR DESCRIPTION
**Pull request by tewshi**
part of #6862

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/history/events/HistoryEventsList.vue [hunk 1]
```diff
@@ -125,14 +126,33 @@ const [DefineTable, ReuseTable] = createReusableTemplate();
...
+                    variant="text"
+                    color="warning"
+                    icon
+                    @click="emit('show:missing-rule-action', item)"
+                  >
+                    <RuiIcon size="16" name="information-line" />
+                  </RuiButton>
+                </template>
+                {{ t('actions.history_events.missing_rule.title') }}
+              </RuiTooltip>
```
Comment (kelsos) [inline frontend/app/src/components/history/events/HistoryEventsList.vue:154]: I think we can put that to `RowActions`' default slot and avoid the wrapper div.

Comment (tewshi) [reply frontend/app/src/components/history/events/HistoryEventsList.vue:154]: oh okay, cool, will do...

File: frontend/app/src/components/history/events/HistoryEventsView.vue [hunk 2]
```diff
@@ -784,6 +815,14 @@ const includeOnlineEvents: ComputedRef<boolean> = useEmptyOrSome(
       />
 
       <TransactionFormDialog :loading="sectionLoading" />
+
+      <MissingRulesDialog
+        v-model="missingRulesDialog"
```
Comment (kelsos) [inline frontend/app/src/components/history/events/HistoryEventsView.vue:820]: maybe do `v-if` here? how much of this renders when the dialog is not visible?

Comment (tewshi) [reply frontend/app/src/components/history/events/HistoryEventsView.vue:820]: it renders the content, will is if to keep it off the dom unless needed

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**